tests/test_models.py

```diff
@@ -106,67 +88,40 @@ def test__build_uuid_list(client):
-        file_type = 'pdf'
-        file_dict = {}
-        m.get(directory_url, text=content)
-        file_list = models.build_file_dict_remote(directory_url, file_type,
-                                                  file_dict)
-        assert '999' in file_list
-
-
-def test_create_csv_from_list(output_dir):
-    """Test create_csv_from_list function."""
-    list_name = ['123']
-    models.create_csv_from_list(list_name, f'{output_dir}output')
-    with open(f'{output_dir}output.csv') as csvfile:
-        reader = csv.DictReader(csvfile)
-        for row in reader:
-            assert row['id'] == '123'
-
-
-def test_metadata_elems_from_row():
-    """Test metadata_elems_from_row function."""
-    row = {'title': 'Test title'}
-    metadata_elem = models.metadata_elems_from_row(row, 'dc.title', 'title',
-                                                   'en_US')
-    assert metadata_elem[0]['key'] == 'dc.title'
-    assert metadata_elem[0]['value'] == 'Test title'
-    assert metadata_elem[0]['language'] == 'en_US'
-    metadata_elem = models.metadata_elems_from_row(row, 'dc.title', 'title')
-    assert metadata_elem[0]['key'] == 'dc.title'
-    assert metadata_elem[0]['value'] == 'Test title'
-    assert 'language' not in metadata_elem[0]
-    row = {'title': ''}
-    metadata_elem = models.metadata_elems_from_row(row, 'dc.title', 'title')
-    assert metadata_elem == []
-    row = {'title': 'Test title 1|Test title 2'}
-    metadata_elem = models.metadata_elems_from_row(row, 'dc.title', 'title',
-                                                   'en_US', '|')
-    assert metadata_elem[0]['key'] == 'dc.title'
-    assert metadata_elem[0]['value'] == 'Test title 1'
-    assert metadata_elem[0]['language'] == 'en_US'
-    assert metadata_elem[1]['key'] == 'dc.title'
-    assert metadata_elem[1]['value'] == 'Test title 2'
-    assert metadata_elem[1]['language'] == 'en_US'
-
-
-# def test_create_ingest_report():
-#     assert False
-
-
-def test_create_metadata_rec():
-    metadata_rec = []
-    row = {'title': 'Test title'}
-    mapping_dict = {'dc.title': ['title']}
-    metadata_rec = models.create_metadata_rec(mapping_dict, row, metadata_rec)
-    assert metadata_rec[0]['key'] == 'dc.title'
-    assert metadata_rec[0]['value'] == 'Test title'
+def test_collection_from_csv(aspace_delimited_csv, aspace_mapping):
+    collection = models.Collection.from_csv(
+        aspace_delimited_csv, aspace_mapping
+        )
+    assert 2 == len(collection.items)
+
+
+def test_collection_post_items(client, input_dir, aspace_delimited_csv,
+                               aspace_mapping):
+    collection = models.Collection.from_csv(
+        aspace_delimited_csv, aspace_mapping
+        )
+    collection.uuid = 'c3d4'
+    items = collection.post_items(client)
+    for item in items:
+        assert item.handle == '222.2222'
+        assert item.uuid == 'e5f6'
+
+
+def test_item_bitstreams_from_directory(input_dir):
```

Comment:
Since having these sorted by name is important, I would confirm that the expected bitstreams are in fact all there in the correct order.